pkg/kp/rcstore/rc.go

```diff
@@ -0,0 +1,95 @@
+package rcstore
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+)
+
+type ReplicationControllerDesires struct {
+	replicasDesired int
+	disabled        bool
+}
+
+type ReplicationControllerData struct {
+	// It is expected that these fields will never be changed after an RC is created.
+	id           string
+	manifest     *pods.Manifest
+	nodeSelector labels.Selector
+	podLabels    labels.Set
+
+	watcher desireWatcher
+
+	// It is expected that these fields can be changed.
+	ReplicationControllerDesires
```

Comment:
Sure, I can assume that any fields can change except for ID. I think for some internal things to work, I still need to have a separate struct called `rcFields` (so that the watcher can still send updates), but I'll no longer export it.